### PR TITLE
A list of minor things pointed out by clang-tidy.

### PR DESCRIPTION
**Pull request by clalancette**
There should be no functional change here.

Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

@mabelzhang There should be no functional changes here, but a quick test on real hardware before merging is probably a good idea.

### CODE REVIEW

**Comment by clalancette** [comment]
@mabelzhang FYI, I tested this out on the actual hardware today with both Dashing and Eloquent.  Everything seemed to work as usual.  So I think this is ready for review and merge.

File: hri_safety_sense/include/hri_safety_sense/MsgHandler.hpp [hunk 1]
```diff
@@ -27,14 +27,16 @@ namespace hri_safety_sense {
  */
 class MsgHandler {
    public:
-	  virtual ~MsgHandler() {};
+          MsgHandler() = default;
+	  virtual ~MsgHandler() = default;
```
Comment (mabelzhang) [inline hri_safety_sense/include/hri_safety_sense/MsgHandler.hpp:31]: Is this line's indentation deliberate?

**Review by mabelzhang — COMMENTED** [review]
I don't really know what the constants related to the file descriptor do, so I'll trust clang-tidy on those... One minor comment. Everything else look fine.

**Comment by clalancette** [comment]
> I don't really know what the constants related to the file descriptor do, so I'll trust clang-tidy on those...

By long-time Unix default, any file descriptors that are open get automatically cloned to a new process when `fork()` is called.  This is both a feature and a bug; it's a cheap way to share file descriptors between processes, but it can also lead to bugs or leaked file descriptors.  Adding `O_CLOEXEC` means to automatically close the file descriptor if the `fork()` is followed by an `exec()`.  In this particular case, it really doesn't do anything since we don't have any calls to `fork()/exec()`, but it is more correct in general and doesn't hurt anything.

File: hri_safety_sense/include/hri_safety_sense/MsgHandler.hpp (hunk 1 above)
Comment (clalancette) [reply hri_safety_sense/include/hri_safety_sense/MsgHandler.hpp:31]: Tabs vs. spaces.  I've now fixed it to be tabs again.

**Pull request merged**